eq-author/src/App/index.js

```diff
@@ -21,34 +22,36 @@ import NotFoundPage from "./NotFoundPage";
 import ErrorBoundary from "./ErrorBoundary";
 
 export const Routes = ({ isSignedIn, ...otherProps }) => (
-  <Router {...otherProps}>
-    <ErrorBoundary>
-      <Switch>
-        <Route path={RoutePaths.SIGN_IN} component={SignInPage} exact />
-        <PrivateRoute
-          path={RoutePaths.HOME}
-          component={QuestionnairesPage}
-          isSignedIn={isSignedIn}
-          exact
-        />
-        <RedirectRoute
-          from="/questionnaire/:questionnaireId/design/:sectionId/:pageId"
-          to={"/q/:questionnaireId/page/:pageId/design"}
-        />
-        <RedirectRoute
-          from="/questionnaire/:questionnaireId/design/:sectionId"
-          to={"/q/:questionnaireId/section/:sectionId/design"}
-        />
-        <PrivateRoute
-          path={RoutePaths.QUESTIONNAIRE}
-          exact={false}
-          component={QuestionnaireDesignPage}
-          isSignedIn={isSignedIn}
-        />
-        <Route path="*" component={NotFoundPage} exact />
-      </Switch>
-    </ErrorBoundary>
-  </Router>
+  <ErrorBoundary>
```

Comment:
This has stopped ErrorBoundary from catching a non-existent questionnaire. Not sure why. Does it need to be within Router?

Reply:
Good spot. It was caused by the `ErrorBoundary` rendering pages with Links on that need to be wrapped in `Router`